webpack.webext.babel.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import config, { src, dist } from './webpack.common';
+import pkg from './package.json';
+
+const variant = process.env.VARIANT; // 'chrome', 'firefox' or 'opera'
```

Comment:
shouldn't this comment contain ``firefox-local`

Reply:
but I guess not so important.

Reply:
Good point. Updated it and added a bit of additional explanation 👍